graviti/schema/param.py

```diff
@@ -116,6 +132,9 @@ def check(self, arg: Any) -> Any:
             ValueError: Raise when the argument is not in options.
 
         """
+        if not arg and self.ptype == PTYPE.Fields:
+            return Fields()
+
         if arg == self.default:
```

Comment:
shall we call `is_default` here?

Reply:
can we check like this?
```python
if self.ptype != PTYPE.Fields and arg == self.default:
```
so that the None value will pass the default check and be sent into `ptype.check`

Reply:
We need to know the `ptype` to return the value. And no need to check the default value. So `is_default` is not called here.